testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabUserHelper.java

```diff
@@ -146,6 +226,19 @@ public static String loadTempUser(final String userType) {
         return tempUser.getUpn();
     }
 
+    public static TempUser loadTempUserForTest(final String userType, Consumer<TempUser> consumer) {
```

Comment:
Synced offline...let's make sure if this `Consumer` is needed...if not, then let's remove it

Reply:
If I'm returning the reference, I don't need it.